### DISCUSSION TITLE
cleaning up files after simulation

### DISCUSSION THREAD

**Discussion by mvanriet** [nunobrum/PyLTSpice#71] [Ideas]
If the results of a simulation are processed automatically, the raw-files are often not needed any more.
So a function like delete_simulation_files() or so could be handy.

**Comment by nunobrum** [comment]
This feature will be integrated in library in the near future.
Creating an issue to track this development.

**Comment by nunobrum** [comment]
An working implementation of this feature is now on 'dev_space' branch.
Check the file_cleanup() method of the SimCommander class.

**Comment by nunobrum** [comment]
Integrated on the main branch. Closing this discussion.
